spec/support/with_project_cleanup.cr

```diff
@@ -0,0 +1,18 @@
+module WithProjectCleanup
+  private def with_project_cleanup(project_directory = "test-project", skip_db_drop = false)
+    yield
+
+    FileUtils.cd(project_directory) {
+      output = IO::Memory.new
+      Process.run(
+        "lucky db.drop",
+        output: output,
+        shell: true
+      )
+
+      output.gets_to_end.should contain("Done dropping")
```

Comment:
What about checking the process output as well this might increase the confidence of this spec. as referenced here https://github.com/luckyframework/breeze/pull/45/files#r633110386

```suggestion
      drop_status = Process.run(
        "lucky db.drop",
        output: output,
        shell: true
      )

      drop_status.normal_exit?.should eq(true)
      output.gets_to_end.should contain("Done dropping")
```

Reply:
I guess you could add a formatter like this.

```crystal
struct NoticeFormat < Log::StaticFormatter
  def run
    message
  end
end

Log.setup(:debug, Log::IOBackend.new(formatter: NoticeFormat))
```

Reply:
Yeah, I think checking the status might be good. I need to figure out why it doesn't run though. The actual `lucky db.drop` never gets called for some reason... I'm not sure why not 🤷‍♂️